nemo/collections/nlp/models/language_modeling/megatron_gpt_sft_model.py

```diff
@@ -779,27 +809,9 @@ def setup_eval_dataloader(self, datasets, data_cfg):
             dataloaders.append(eval_dl)
         return dataloaders
 
-    def _reset_activation_checkpointing_args(self):
-        if self.cfg.get('megatron_amp_O2', False):
-            base_module = self.model.module
-        else:
-            base_module = self.model
-
-        base_module.language_model.encoder.activations_checkpoint_granularity = None
-        base_module.language_model.encoder.activations_checkpoint_method = None
-        base_module.language_model.encoder.activations_checkpoint_num_layers = None
-
-    def _restore_activation_checkpointing_args(self):
-        if self.cfg.get('megatron_amp_O2', False):
-            base_module = self.model.module
-        else:
-            base_module = self.model
-        base_module.language_model.encoder.activations_checkpoint_granularity = self.original_checkpointing_granularity
-        base_module.language_model.encoder.activations_checkpoint_method = self.original_checkpointing_method
-        base_module.language_model.encoder.activations_checkpoint_num_layers = self.original_checkpointing_num_layers
-
     def on_validation_epoch_start(self):
         self._reset_activation_checkpointing_args()
+        self._reset_sequence_parallelism_args()
```

Comment:
where is the restore activation checkpointing and sequence parallel ?

Reply:
These two functions are inherited from `MegatronGPTModel`